packages/scandipwa/src/component/VideoPopup/VideoPopup.style.scss

```diff
@@ -12,12 +12,15 @@
 .VideoPopup {
     --popup-min-width: none;
     --popup-content-padding: 0;
+    --content-wrapper-width: none;
+    --popup-max-height: none;
```

Comment:
is It used anywhere ?

Reply:
--content-wrapper-width is used in multiple places and was limiting the width of video popup.
--popup-max-height was made by me to remove popup height limitation from video popup's style file.
Now that i'm thinking about it, it's probably not the best practice, i'm open to suggestions

Reply:
Hm, I think if it doesn't break anything, then seems legit for me.